internal/storage/rocksdb/store.go

```diff
@@ -161,6 +199,21 @@ func (rdb *rocksDB) Close() error {
 	return nil
 }
 
+func (rdb *rocksDB) PutTTL(key []byte, value []byte, expireTS int64) error {
+	defer rdb.opts.statsCli.Timing("rocksdb.putTTL.latency.ms", time.Now())
```

Comment:
Please replace `time.Now()` with `hlc.Now()`. Also elsewhere in the whole storage layer.

Reply:
Let's do that in a separate PR.